Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1094,21 +1087,25 @@ def audit_version
         end
       end
 
+      # TODO: check could be in RuboCop
       problem "version #{version} should not have a leading 'v'" if version.to_s.start_with?("v")
 
       return unless version.to_s.match?(/_\d+$/)
 
+      # TODO: check could be in RuboCop
       problem "version #{version} should not end with an underline and a number"
     end
 
     def audit_download_strategy
       if url =~ %r{^(cvs|bzr|hg|fossil)://} || url =~ %r{^(svn)\+http://}
+        # TODO: check could be in RuboCop
         problem "Use of the #{$&} scheme is deprecated, pass `:using => :#{Regexp.last_match(1)}` instead"
       end
 
       url_strategy = DownloadStrategyDetector.detect(url)
 
       if using == :git || url_strategy == GitDownloadStrategy
```

Comment:
@MikeMcQuaid could you give me a hint on how to access `using` and `specs` in `Rubocop`? I was trying to figure it out myself, but since I'm not super familiar with `Rubocop` and the node helpers implemented in `../rubocops/extend/formula.rb` I could need a helping hand. Thx.

Reply:
This `# TODO` is incorrect, actually; it requires a bit too much of Homebrew's internals. I'll remove it, thanks 👍

Reply:
(there may well be other incorrect ones as I am but a fallible human, feel free to shout/ask if you see any others)